package.json

```
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/rivethead42/hello-node#readme",
   "engines": {
-    "node": "8.9.4",
-    "npm": "5.6.0"
+    "node": "8.16.0",
```

We could also specify the npm update type, so that this is directly tied to an exact version of node.  This way, if 8.16.0 is no longer supported but there is another 8.x version supported, this would still work with no changes.
```suggestion
    "node": "^8.16.0",
```